symplyphysics/laws/astronomy/lifetime_of_star_on_main_sequence.py

```diff
@@ -0,0 +1,43 @@
+from sympy import (Eq, solve, S)
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+    convert_to,
+    dimensionless,
+)
+
+# Description
+## The main sequence is the stage of stellar evolution formed by the stars at this stage and the corresponding luminosity class.
+## The main sequence of stars falls after the protostar stage. At this point, the age of the star is considered zero and it is
+## located on the so-called initial main sequence.
+
+## Law is: T = 10 * M^(1 - n), where
+## T - lifetime on the main sequence in billions of years,
+## M - mass of the star (in masses of the Sun),
+## n - indicator depending on the mass of the star.
+
+lifetime = Symbol("lifetime", dimensionless)
+
+mass_of_star = Symbol("mass_of_star", dimensionless)
```

Comment:
Use proper dimensions and divide by respective quantities.

Reply:
> Use proper dimensions and divide by respective quantities.

It doesn't work that way. That's why.

I did it like this:
![image](https://github.com/blackyblack/symplyphysics/assets/100017508/b4aff2cf-76de-437f-9201-dbbe02387acb)
![image](https://github.com/blackyblack/symplyphysics/assets/100017508/eb61de6e-6e31-4a4b-8add-53b2cba0c435)

I looked at what result_expr looks like:
![image](https://github.com/blackyblack/symplyphysics/assets/100017508/2bfb2bf5-233c-4d77-bee0-959e28319bd0)
QTY1 - mass_of_sun = Quantity(1.989e30 * units.kilogram).

I can assume that because of this kind of result_expr, this is the output value:
![image](https://github.com/blackyblack/symplyphysics/assets/100017508/0411dff6-9767-4200-ab4b-f15a7c02b39c)

Because the same value is raised to the power of "n-1" and to the power of "1 - n", and only then they multiply by each other. When raising to a power, values that are not ideally inverse to each other are obtained, so I get this answer.
Or I'm wrong and the problem is something else.

Reply:
You've got correct results. But you've got the lifetime equal to 10 billion years and compare it to 1 year.